jax/api_util.py

```diff
@@ -74,16 +75,78 @@ def flatten_fun_nokwargs2(in_tree, *args_flat):
   aux_flat, aux_tree = tree_flatten(aux)
   yield (ans_flat, aux_flat), (ans_tree, aux_tree)
 
-def argnums_partial(f, dyn_argnums, args):
+
+def argnums_partial(f, dyn_argnums, args, raise_error=False):
+  """Keeps only dynamic arguments for the arguments of f.
+
+  Args:
+    f: The function being wrapped.
+    dync_arguments: The indices of the dynamic arguments (i.e. non-static
+      positial arguments).
+    args: The positional arguments.
+
+  Returns:
+    A tuple of:
+      The function wrapped into a partial with the non dynamic arguments
```

Comment:
This docstring isn't quite accurate (first element of the return value isn't a partial, it's an lu.WrappedFun), and moreover I don't think this docstring conveys more than type annotations.

Can you revert the changes to `argnums_partial`, both in the code and in the docstring? IIUC it doesn't need to change in this PR.

Reply:
Done.